src/main/java/jenkins/branch/OrganizationFolder.java

```diff
@@ -173,7 +173,8 @@ public void complete() throws IllegalStateException, InterruptedException {
                             for (MultiBranchProjectFactory factory : projectFactories) {
                                 MultiBranchProject<?, ?> project;
                                 try {
-                                    project = factory.createProject(OrganizationFolder.this, projectName, sources, Collections.<String,Object>emptyMap(), listener);
+                                    project = factory.createProject(OrganizationFolder.this, projectName, sources, getOrphanedItemStrategy(),
```

Comment:
> The idea was to resolve an urgent problem I found in several production environment.

The change involves API additions, if not settings format changes, so it is not eligible for a hotfix. It has to be done right the first time.